crates/sui-graphql-rpc/src/lib.rs

```diff
@@ -17,3 +17,4 @@ pub(crate) mod raw_query;
 pub mod server;
 pub mod test_infra;
 mod types;
+pub(crate) mod zklogin_verify_signature;
```

Comment:
nit: we should instead route this through https://github.com/MystenLabs/sui/blob/main/crates/sui-graphql-rpc/src/types/mod.rs

Reply:
revised